source/_templates/footer.html

```diff
@@ -18,7 +18,7 @@
         {%- if hasdoc('copyright') %}
         {% trans path=pathto('copyright'), copyright=copyright|e %}<a href="{{ path }}">Copyright</a> &copy; {{ copyright }}.{% endtrans %}
         {%- else %}
-        {% trans copyright=copyright|e %}Copyright &copy; 2020 Hyperledger {% endtrans %}
+        {% trans copyright=copyright|e %}Copyright &copy; 2021 Perun Framework Contributors {% endtrans %}
```

Comment:
```suggestion
        {% trans copyright=copyright|e %}Copyright &copy; 2020 Perun Framework Contributors {% endtrans %}
```
or
```suggestion
        {% trans copyright=copyright|e %}Copyright &copy; 2020, 2021 Perun Framework Contributors {% endtrans %}
```

Reply:
OK, changed the copyright year to 2020-2021.